test/computational_graph.jl

```diff
@@ -108,20 +132,30 @@ end
     @test external_labels(g2) == [1, 4, 5, 3, 2]
 end
 
+@testset_skip "TODO: Update to new API" "Graph Operations" begin
```

Comment:
why you need new API for this test?

Reply:
It uses the old interaction and feynman_diagram format. I will update it now.

Reply:
Fixed